src/crystal/dwarf/info.cr

```diff
@@ -67,19 +67,19 @@ module Crystal
           end
         when FORM::Block1
           len = @io.read_byte.not_nil!
-          @io.read_fully(bytes = Bytes.new(len.to_i))
+          @io.read_fully(bytes = Bytes.new(len.to_i!))
```

Comment:
Why the unsafe conversion is used in this operations?

Reply:
Because I thought `to_i!` was the one that would raise on overflow but I later remember it's the other way around. In general you can consider all my changes "quick and dirty to make it work". Once we decide we want to go with this I'll review the PR from scratch and change any `to_i!` to `to_i`, and also do some other adjustments (or we can do them all in parallel).